source/js/common/index.js

```diff
@@ -0,0 +1,8 @@
+// The following modules are the common modules which need to be called
+// on all pages on the main Foundation site, apps, and microsites.
+// e.g., Foundation site, PNI, MozFest etc
+
+export { default as bindEventHandlers } from "./bind-event-handlers";
```

Comment:
if we're doing this, we might as well do proper named import/export and then do `export { NamedThing } from "module";` pass-through here.